### ISSUE TITLE
Need to build review and commenting code

### ISSUE THREAD

**Issue by peller** [maqetta/maqetta#60]
The easiest thing would be to just fold the davinci.review project into davinci.core, since there's no obvious way to do a build across projects.  Otherwise, we'd have to stage an area where we copy one over the other.

Should we build a single layer for all the review code, or perhaps a separate layer for the part which is used in the workbench?

We might also want to reconsider having the review tab on the workbench, since it pulls in some of the review code at startup.

**Comment by JonFerraiolo** [comment]
@peller, @jhpedemonte

Can we close this bug or at least defer it to M5? Won't the review/commenting code get built automatically when the R/C code is integrated into the core?

**Comment by peller** [comment]
yes.  Pushing to M5.

**Comment by waynevicknair** [comment]
we are currently building the review and commenting code as evidenced by it's presence as a feature in the running version of maqetta. review and commenting will continue to be built once the core integration is merged into master.

**Comment by JonFerraiolo** [comment]
Actually, what we mean by "build" is running the dojo builder which compacts the code by removing white space, merging JS files into a single file, and things like that, and we don't currently do it with the R/C files. I know this for sure because this weekend I was debugging into those files when running a Maqetta build. (It was actually quite handy that they weren't "built" - I was able to fix bugs quickly)

**Comment by waynevicknair** [comment]
oh. well that's completely different. :)

**Comment by peller** [comment]
@JonFerraiolo: it's not so hard to debug with a build if shrinksafe is turned off... is it worth generating builds with this option?  Perhaps it's something we could have done "on demand"?

**Comment by peller** [comment]
let's try to build late-bound R&C code (e.g. the view and dependencies) into a separate layer for M6.  Will require factoring as much code away from pagedesigner/Workbench/Runtime as possible.  See #1670

**Comment by JonFerraiolo** [comment]
-> @peller - is this issue still valid?

**Comment by peller** [comment]
Yes, the Workplace dependencies on Review (CommentView and friends) are fewer than they once were, but the files in CommentView are still uncompressed and unbuilt.  CommentView should be built as a separate layer, CommentExplorerView taken out of davinci and built as a layer, too, and we should consider whether loose .js modules should be processed for whitespace and compressed.

**Comment by JonFerraiolo** [comment]
I'm inclined to push this to M7 Prio High - too high of risk at too low benefit to try to fix for M6 at this point.

**Comment by peller** [comment]
I think this is fixed now.
